libcaliptra/examples/generic/test.h

```diff
@@ -0,0 +1,24 @@
+// Licensed under the Apache-2.0 license
+#pragma once
+
+#include <stdint.h>
+
+#include "caliptra_types.h"
+
+typedef struct test_info {
+  struct caliptra_buffer rom;
+  struct caliptra_buffer image_bundle;
+  struct caliptra_fuses fuses;
```

Comment:
Thanks for adding this. The uds_seed and field_entropy are actually in that caliptra_fuses struct though. So you probably won't want to duplicate them here.

Reply:
Oh shoot, thanks! That would have been really redundant.